streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -93,8 +92,7 @@
 
     private static final String WINDOWED_NAME = "KSTREAM-WINDOWED-";
 
-    private static final LongSerializer LONG_SERIALIZER = new LongSerializer();
-    private static final LongDeserializer LONG_DESERIALIZER = new LongDeserializer();
+    private static final SerDe<Long> LONG_SER_DE = SerDes.LONG();
```

Comment:
Yeah, my comment is based under the assumption that SerDes.LONG() will return a constant (I can't think of a reason why it should not). Even the method name capitalisation suggests this.